[Bug]: Cmake serch path for libdivecomputer does not match submodule configuration

### Description

FindLibdivecomputer.cmake search path for include and librairies assumes subsurface and libdivecomputer are cloned in the same parent folder, which does not match the gitmodules settings that create a libdivecomputer under the subsurface folder.

### Subsurface Version

6008bfe9276f5f69ae509ca70bb30c3c8363538b

### Platform(s)

Linux

### Platform Version

Fedora 40

### Divecomputer Model

_No response_

### Steps to Reproduce

`git clone https://github.com/subsurface/subsurface.git`
`cd subsurface`
`git submodule update --init --recursive`
`cd libdivecomputer`
`autoreconf --install`
`./configure`
`make`
`cd ..`
`mkdir build`
`cd build`
`cmake ..`

Then come the error:
`CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find Libdivecomputer (missing: LIBDIVECOMPUTER_INCLUDE_DIR
  LIBDIVECOMPUTER_LIBRARIES)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  cmake/Modules/FindLibdivecomputer.cmake:39 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  cmake/Modules/HandleFindLibDiveComputer.cmake:5 (find_package)
  CMakeLists.txt:149 (include)
`


### Logs

```shell

```

### Mentions

_No response_

--- a/cmake/Modules/FindLibdivecomputer.cmake
+++ b/cmake/Modules/FindLibdivecomputer.cmake
@@ -19,7 +19,7 @@ NAMES
     libdivecomputer/descriptor.h
 HINTS
     ${CMAKE_CURRENT_SOURCE_DIR}/../install-root/include
-    ${CMAKE_CURRENT_SOURCE_DIR}/../libdivecomputer/include/
+    ${CMAKE_CURRENT_SOURCE_DIR}/libdivecomputer/include/
 PATHS
     /opt/out/include
 )
@@ -27,10 +27,9 @@ PATHS
 FIND_LIBRARY( LIBDIVECOMPUTER_LIBRARIES
 NAMES
     libdivecomputer.a
-    divecomputer
 HINTS
     ${CMAKE_CURRENT_SOURCE_DIR}/../install-root/lib
-    ${CMAKE_CURRENT_SOURCE_DIR}/../libdivecomputer/src/.libs/
+    ${CMAKE_CURRENT_SOURCE_DIR}/libdivecomputer/src/.libs/
 PATHS
     /opt/out/lib
 )

